api_server.py

```
@@ -229,37 +250,53 @@ def health():
 def root():
     try:
         now = datetime.now()
```

Ruff (0.15.7)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)